DevoxxClientMobile/src/main/java/com/devoxx/views/helper/Util.java

```diff
@@ -114,6 +115,28 @@ public static FloatingActionButton createWebLaunchFAB(Supplier<String> urlSuppli
             });
         });
     }
+    
+    public static void requestRating() {
+        String url = null;
+        if(Platform.isAndroid()) {
+            String packageName = "com.devoxx";
+            url = "market://details?id=" + packageName;
+        } else if (Platform.isIOS()) {
+            String appId = "1094805620";
+            url = "itms-apps://itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?id=" + appId +"&amp;onlyLatestVersion=true&amp;pageNumber=0&amp;sortOrdering=1&amp;type=Purple+Software";
```

Comment:
This doesn't work on iOS 11. See: 
https://stackoverflow.com/a/46335171/3956070
This works for me:
`url = "itms-apps://itunes.apple.com/app/itunes-u/id" + appId +"?action=write-review";`

Reply:
The predicament is that there is no way in Charm-Down to know the version of platform. And since I am not sure if the new version works with all previous versions of iOS, I went with the most popular one.

Reply:
Apple [stats](https://developer.apple.com/support/app-store/) indicate that iOS 11 has 81% adoption by May 2018, so "popular" one won't work here :)